core/src/com/unciv/ui/components/fonts/DiacriticSupport.kt

```diff
@@ -54,7 +54,7 @@ object DiacriticSupport {
 
     private class LineData(capacity: Int) {
         val output = StringBuilder(capacity)
-        val accumulator = StringBuilder(8)
+        val accumulator = StringBuilder(9) // touhidurrr said there can be nine
```

Comment:
kind of hilarious that you would phrase it like this. i pointed out that it can be more, not necessarily 9. the wiki link maximum is based on the number of diacritics, not based on the number of total characters with diacritics.
anyways, if it can grow then 8 or 9 can be a good enough initial space.